experimental/uefi/loader/src/main.rs

```diff
@@ -27,19 +26,29 @@ use tokio_util::codec::Decoder;
 mod qemu;
 mod server;
 
+#[derive(clap::ArgEnum, Clone, Debug, PartialEq)]
+enum Mode {
+    Uefi,
+    Bios,
+}
+
 #[derive(Parser, Debug)]
 struct Args {
     /// path to the `qemu-system-x86_64` binary
     #[clap(long, parse(from_os_str), validator = path_exists, default_value_os_t = PathBuf::from("/usr/bin/qemu-system-x86_64"))]
     qemu: PathBuf,
 
+    /// execute either in UEFI mode or BIOS mode
```

Comment:
nit: (some pre-exisitng) comments should start with a capital letter and end with a `.`